jquery/exercise3/js/manipulate.js

```diff
@@ -0,0 +1,11 @@
+function Manipulate(argument) {
+  $("#myList").append("<li></li>");
+  $("#myList>li").filter(":odd").remove();
+  $("div:.module").last().append("<h2></h2>","<p></p>");
+  $("select").append("<option>wednesday</option>");
+  $("select>option").last().attr("value","wednesday");
```

Comment:
it should be writeen as selector1 space selector2
add spaces before and after operator
select > option
div: .module